tests/queries/0_stateless/01233_check_table_with_metadata_cache.sh

```diff
@@ -12,6 +12,13 @@ database_engines=(Ordinary Atomic)
 use_metadata_caches=(false true)
 use_projections=(false true)
 
+function materialize_projection_if_needed()
+{
+    if [[ "$use_projection" == "true" ]]; then
+        ${CLICKHOUSE_CLIENT} --query "ALTER TABLE test_metadata_cache.check_part_metadata_cache MATERIALIZE PROJECTION p1"
```

Comment:
`CHECK TABLE` should not fail w/o projection materialization I guess, what do you think?

Reply:
Yes. Adding projection materialization can't fix this test, so it is useless.